src/SearchByTitleForm/SearchByTitleForm.js

```diff
@@ -36,18 +40,10 @@ class SearchByNameForm extends Component {
                     value={this.state.movieTitle} 
                     name='movieTitle' 
                     placeholder='Movie Name' 
-                    onChange={(event) => this.handelMovieTitle(event)}
+                    onChange={this.handelMovieTitle}
                 />
-                <Link to={'/' + this.state.id} >
-                    <input 
-                        className='search-by-title-submit'
-                        type='submit'
-                        value='SEARCH'
-                        disabled={!this.state.movieTitle.length}
-                        onClick={(event => this.handelSubmit(event))}
-                    />
-                </Link>
-
+                <NavLink className='search-by-title-submit' to={`/${this.props.id}`} onClick={(event) => this.handelSubmit(event)}
```

Comment:
Not sure how this is interacting with Redirect actually but onClick is at least doing something